test/cat.jl

```diff
@@ -120,11 +120,11 @@ module TestCat
     @test size(vcat(df, df, df)) == (size(df, 1) * 3, size(df, 2))
 
     alt_df = deepcopy(df)
-    vcat(df, alt_df)
+    @test size(vcat(df, alt_df)) == (8, 3)
```

Comment:
It would be even better to check the actual result, right? It's weird that the existing code didn't...

Reply:
added (I have added a similar test earlier elsewhere also).